### ISSUE TITLE
Using two on the same page causes error

### ISSUE THREAD

**Issue by arfath77** [kfrancikowski/react-multiselect-dropdown-bootstrap#15]
When we use two drop down on same page and select different options on both of them, then both of them shows the same value, added the below code to fix it.

componentDidUpdate(prevProps) {
        const { selected } = this.props;
        if (prevProps.selected !== selected) this.setState({ selected });
    }

**Comment by lanepemberton** [comment]
@arfath77 This is not an issue and should be closed. All you need to do is assign unique names to the component OR if you're instantiating them in a list, pass the index of the loop into the component and append to the name property.

**Comment by vicreinhart** [comment]
I agree this is not an issue.  I have two components on the same page and it does work for me.

**Comment by etekumoses** [comment]
was this fixed i cant deploy with this error
